WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/details/ThreatDetailsNavigationEvents.kt

```diff
@@ -17,4 +17,8 @@ sealed class ThreatDetailsNavigationEvents {
     data class ShowUpdatedScanStateWithMessage(@StringRes val messageRes: Int) : ThreatDetailsNavigationEvents()
 
     data class ShowUpdatedFixState(val threatId: Long) : ThreatDetailsNavigationEvents()
+
+    object ShowGetFreeEstimate : ThreatDetailsNavigationEvents() {
+        const val url = "https://codeable.io/partners/jetpack-scan/"
```

Comment:
❓ - Do we have an alternative for hardcoding or is this sufficient for the MVP? Post MVP 🤔  perhaps an endpoint similar to jetpack capabilities, but for upsells?

Reply:
I asked the same question yesterday on Slack - tbh I'm not sure. I don't like hardcoding urls - especially 3rd party. They can change the url and we'll never be able to remove it from this version of the app. 
I was thinking if we could for example point to our url and redirect from there. Downsides of this approach is that some browsers might complain and suppress the redirect to a different domain.

Reply:
I'll merge this PR and update the code if we make a different decision on Slack.